sbin/patchman

```diff
@@ -579,6 +591,9 @@ def process_args(args):
     if args.errata:
         update_errata(args.force)
         showhelp = False
+    if args.delete_host:
+        host_delete(args.delete_host)
```

Comment:
I think this should be:

```
if args.host_delete:
    host_delete(args.host)
```

That would be consistent with the style of the other commands that take hosts as arguments.

Reply:
So you want it to be -k  -H host ?  wouldn't that mean if you just ran -k it would delete them all?  Everything else works on all if -H isn't given, but for this we don't want that behavior at all.

Reply:
Oh, you're right. That's probably not good default behaviour! Maybe it should _require_ the `-H` argument?

Reply:
Well that's why I have the -k argument as being "different", because it's a little more..  dangerous maybe?  Nothing else patchman does "breaks" things, so I'm kinda ok with you having to read the help a bit..

In theory we could change it back around to having the hosts none check in the function and then requiring -H, but with the way the code is currently written the safety checks all work as designed.  if you do patchman -k<enter>  it properly complains, there really isn't an easy way to delete every host, which is good IMHO, and I've tested it pretty extensively as-is.

This is one of the patches in my local tree that I use pretty much daily in my workflow, as I have fully automated ansible jobs reaching out to it for server decoms.